packages/overlay/test/renderer.test.js

```diff
@@ -1,114 +1,147 @@
 import { expect } from '@esm-bundle/chai';
-import { fixtureSync } from '@vaadin/testing-helpers';
+import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import '../vaadin-overlay.js';
 
 describe('renderer', () => {
-  let rendererContent, renderModel;
+  let overlay, content;
 
-  beforeEach(() => {
-    rendererContent = document.createElement('p');
-    rendererContent.textContent = 'renderer-content';
-    renderModel = { selected: true };
+  beforeEach(async () => {
+    overlay = fixtureSync('<vaadin-overlay></vaadin-overlay>');
+    await nextRender();
+    content = document.createElement('p');
+    content.textContent = 'renderer-content';
   });
 
-  describe('without overlay', () => {
```

Comment:
Nested suite is no longer needed, it's a leftover from the [V14 version](https://github.com/vaadin/vaadin-overlay/blob/a02120cd6d651019181a23b7cb7f19d5864da6ba/test/renderer.html#L41) that supported `<template>`.